### DISCUSSION TITLE
leetcode-js/blog/go_home

### DISCUSSION THREAD

**Discussion by giscus** [2xiao/leetcode-js#3] [Announcements]
# leetcode-js/blog/go_home

回到小镇的一些感受 国庆回了趟家，讲一讲回家后的一些感受。 第一个感受是回了家，世界的游戏规则和价值判断好像就变了。 在十八线小镇，挣钱是特别困难的，再努力也没用，换句话说，就是蛋糕太小了。

https://2xiao.github.io/leetcode-js/blog/go_home.html

**Comment by botsp** [comment]
浮云游子意、落日故人情
